CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to
 
 ## Unreleased
 
```

Comment:
8.6.3?

Reply:
Was going to add it in with the next `lerna version`. It's not a big thing, but if someone else merged changes then that might affect the version number.

Reply:
Nah, my bad!